runscript.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import unittest
+import xmlrunner
+import inspect
+from module.cliparser import build_parser
+from module.buildcap import build_capabilities
+from selenium import webdriver
+
+# exit codes
+ERROR = 3
+UNSTABLE = 2
+FAILED = 1
+PASSED = 0
+ret = PASSED
```

Comment:
How about EXEC_FAILED, EXEC_ERROR, TEST_FAILED, TEST_ERROR to make it easy for caller to distinguish?